src/event/messages/nez.ts

```diff
@@ -4,6 +4,8 @@ import { NOSE } from '../../config'
 import zenCountSchema from '../../schemas/zenCountSchema'
 import { isMirrorTime } from '../../utils'
 
+const MAX_TIME_DIFFERENCE_HOURS = 1.5
```

Comment:
We should put everything in ms or add a new dep that handle time in a cleaner way
otherwise it will lead to inaccuracies